t/38-pg-connection-lock.t

```diff
@@ -0,0 +1,50 @@
+v6;
+use Test;
+use DBIish;
+
+plan 3;
+
+my %con-parms;
+
+# If env var set, no parameter needed.
+%con-parms<dbname> = 'dbdishtest' unless %*ENV<PGDATABASE>;
+%con-parms<user> = 'postgres' unless %*ENV<PGUSER>;
+%con-parms<port> = 5432;
+# Test for issue #62
+
+my $dbh;
+
+try {
+    $dbh = DBIish.connect('Pg', |%con-parms);
+    CATCH {
+        when X::DBIish::LibraryMissing | X::DBDish::ConnectionFailed {
+            diag "$_\nCan't continue.";
+        }
+        default { .throw; }
+    }
+}
+without $dbh {
+    skip-rest 'prerequisites failed';
+    exit;
+}
+
+is $dbh.is-connection-available(), True, 'Connection is available';
+
+# Use the connection for some activity
+my $sth = $dbh.prepare('SELECT pg_sleep(0.3)');
+my $p1 = start {
+    $sth.execute();
+}
+
+# Ensure the query is running
+sleep 0.05;
```

Comment:
Such short sleeps are a risk on heavily loaded systems (such as blin, which I imagine tests a load of modules in parallel).